src/Public/js/nipsweb.librarypane.js

```diff
@@ -10,7 +10,7 @@ var entityMap = {
   "&": "&amp;",
   "<": "&lt;",
   ">": "&gt;",
-  '"': "&quot;",
+  "\"": "&quot;",
```

Comment:
Huh, I tried this before and my syntax highlighting went weird, but if it works, it works :)